actions/bump-version/action.yaml

```diff
@@ -33,12 +33,16 @@ runs:
     - name: Bump version
       id: bump-version
       run: |
-        parameters=(--no-commit --no-tag ${{ inputs.release-type }})
+        parameters=(--no-commit --allow-dirty --no-tag ${{ inputs.release-type }})
+        test=(${{ inputs.release-type }})
```

Comment:
Not needed anymore?

Reply:
Which one is not needed ?

Reply:
The test variable

Reply:
I removed it and tested in the [repo](https://github.com/bakdata/github-playground-3)